### PR TITLE
Add InfluxDB to datastores

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ toc::[]
 
 * https://calcite.apache.org/[Apache Calcite] - SQL parser, building blocks for datastores.
 * https://druid.apache.org/[Apache Druid] - A high performance real-time analytics database.
+* https://www.influxdata.com/[InfuxDB] - Purpose-Built Open Source Time Series Database.
```
Comment (gunnarmorling) [inline README.adoc]: InfuxDB -> InfluxDB

Could you also move it after Apache Pinot, so to keep it alphabetically sorted? Thanks!

Comment (geoand) [reply README.adoc]: Oops! Sure thing

Comment (geoand) [reply README.adoc]: Done

**Comment by gunnarmorling** [comment]
Nice one, thx!

**Comment by gunnarmorling** [comment]
Datastores is quite sparse still, but I thought I'd put out what I got and see what others come up with. Seems it's the right way :)

**Comment by geoand** [comment]
I'm sure it will fill up pretty soon :wink:

**Pull request merged**